apps/test-stress-can/src/main.cpp

```diff
@@ -0,0 +1,38 @@
+#include "mbed.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+
+CANBus can(CAN_RX, CAN_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANMsg msg;
+
+DigitalOut led(LED1);
+InterruptIn btn(BUTTON_1);
+
+constexpr uint16_t sent = 1000;
+uint16_t received = 0;
+ 
+// create an event queue
+EventQueue queue;
+
+void receiveMessageOutside(CANMsg &msg) {
+  // process msg or something
+  received++;
```

Comment:
maybe for this test app you could have the other can node send incrementing numbers in the msg data and here you can check to make sure its received in the same order it was sent